app/web/utils/date.ts

```diff
@@ -19,24 +19,30 @@ const monthNames = [
   "December",
 ];
 
-const dateTimeFormatter = new Intl.DateTimeFormat(navigator.language, {
-  month: "short",
-  year: "numeric",
-});
+const dateTimeFormatter = new Intl.DateTimeFormat(
+  typeof navigator !== "undefined" ? navigator.language : "en",
```

Comment:
Since we now learnt that Next pass the locale through internationalised routing, might we be able to restructure this so it takes that into consideration? Maybe one for a follow-up PR if you don't want to add to this diff